internal/bminventory/inventory.go

```
@@ -1845,22 +1845,33 @@ func (b *bareMetalInventory) updateNonDhcpNetworkParams(updates map[string]inter
 	}
 
 	var err error
-	*machineCidr, err = network.CalculateMachineNetworkCIDR(apiVip, ingressVip, cluster.Hosts)
+	err = verifyParsableVIPs(apiVip, ingressVip)
 	if err != nil {
-		log.WithError(err).Errorf("failed to calculate machine network cidr for cluster: %s", params.ClusterID)
-		return common.NewApiError(http.StatusBadRequest, err)
+		log.WithError(err).Errorf("Failed validating VIPs of cluster id=%s", params.ClusterID)
+		return err
 	}
+	*machineCidr, _ = network.CalculateMachineNetworkCIDR(apiVip, ingressVip, cluster.Hosts)
```

Why ignore the error?